### ISSUE TITLE
Failed to deploy to npm

### ISSUE THREAD

**Issue by cybertk** [cybertk/ramlev#26]
Failed to deploy to npm for v0.4.0 on [Travis CI](https://travis-ci.org/cybertk/ramlev/jobs/72776265)

```
NPM API key format changed recently. If your deployment fails, check your API key in ~/.npmrc.
http://docs.travis-ci.com/user/deployment/npm/
npm ERR! need auth auth and email required for publishing
npm ERR! need auth You need to authorize this machine using `npm adduser`
npm ERR! System Linux 3.13.0-40-generic
npm ERR! command "/home/travis/.nvm/v0.10.40/bin/node" "/home/travis/.nvm/v0.10.40/bin/npm" "publish"
npm ERR! cwd /home/travis/build/cybertk/ramlev
npm ERR! node -v v0.10.40
npm ERR! npm -v 1.4.28
npm ERR! code ENEEDAUTH
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /home/travis/build/cybertk/ramlev/npm-debug.log
npm ERR! not ok code 0
```

**Comment by cybertk** [comment]
Solved by re-authenticate npm, and get the new **API Token** from `~/.npmrc`

```
rm ~/.npmrc
npm login
```

For registry,

```
npm config set registry https://registry.npm.taobao.org
```

**Comment by cybertk** [comment]
Fixed by 86c0ff79ddd44f3763270707e12015488eac3a10
